wright/src/grammar/parsers/expression/func_call.rs

```diff
@@ -0,0 +1,59 @@
+use crate::grammar::ast::eq::AstEq;
+use crate::grammar::ast::{Expression, FuncCallExpression};
+use crate::grammar::model::{Fragment, HasFragment};
+use crate::grammar::parsers::expression::ToExpression;
+use crate::grammar::parsers::whitespace::token_delimiter;
+use crate::grammar::parsers::with_input;
+use nom::character::complete::char as ch;
+use nom::combinator::map;
+use nom::multi::separated_list;
+use nom::sequence::{delimited, pair, tuple};
+use nom::IResult;
+
+impl<'s> FuncCallExpression<'s> {
+    /// Left parenthesis delimiting argument list. Probably should never change.
+    pub const DELIMITER_LEFT: char = '(';
+
+    /// Left parenthesis delimiting argument list. Probably should never change.
+    pub const DELIMITER_RIGHT: char = ')';
+
+    /// Comma separating arguments. Probably should never change.
+    pub const ARG_SEPARATOR: char = ',';
+
+    /// Parse an index expression in wright source code.
+    pub fn parse(input: Fragment<'s>) -> IResult<Fragment<'s>, Self> {
+        map(
+            with_input(pair(
+                Expression::parse,
```

Comment:
Need something that isn't Expression::parse here;
If someone writes
`!func(args)` we want that to parse as `Not(FuncCall(Name, Args))` not `FuncCall(Not(Name), args)`

Reply:
Fixed